Beta

blackfalcon opened the pull request:
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** # (issue, if applicable)

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)


## Checklist:

- [ ] My update follows the CONTRIBUTING guidelines of this project
- [ ] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Release AuroKit v1.0.0-beta.14: rename the package to `@aurodesignsystem/aurokit`, consolidate and update all Auro component dependencies, and deliver comprehensive documentation and Storybook integration while updating CI workflows to support the new beta branch.

New Features:
- Integrate Storybook with configuration, component stories, and MDX guides for AuroKit

Enhancements:
- Rename package and repository references from `auro-kit` to `aurokit` and bump to version 1.0.0-beta.14
- Update dependencies to latest major/minor releases and require Node.js ^20.x || ^22.x
- Add npm scripts for running and building Storybook

CI:
- Extend GitHub Actions workflow and branch protection to include the beta branch and update test contexts

Documentation:
- Expand README with installation, update policies, override instructions, roadmap, and support guidelines
- Add CHANGELOG.md, update docs/upstream.md, and refresh issue templates to reference `aurokit`

blackfalcon committed:
chore: add icons to dependencies

blackfalcon committed:
perf: add accordion to package.json

semantic-release-bot committed:
chore(release): 1.0.0-beta.1 [skip ci]

# 1.0.0-beta.1 (2024-06-04)

### Performance Improvements

* add accordion to package.json ([40f4069](https://github.com/AlaskaAirlines/auro-kit/commit/40f40699582564e711a6fffad0ed9d2b548c0ec4))

blackfalcon committed:
perf: add remaining Auro elements

blackfalcon committed:
perf: remove @aurodesignsystem/auro-kit from list

semantic-release-bot committed:
chore(release): 1.0.0-beta.2 [skip ci]

# [1.0.0-beta.2](https://github.com/AlaskaAirlines/auro-kit/compare/v1.0.0-beta.1...v1.0.0-beta.2) (2024-06-04)

### Performance Improvements

* add remaining Auro elements ([0e70e02](https://github.com/AlaskaAirlines/auro-kit/commit/0e70e027c9e6f0143a7aa311a75e7a5fc875d914))

semantic-release-bot committed:
chore(release): 1.0.0-beta.3 [skip ci]

# [1.0.0-beta.3](https://github.com/AlaskaAirlines/auro-kit/compare/v1.0.0-beta.2...v1.0.0-beta.3) (2024-06-04)

### Performance Improvements

* remove @aurodesignsystem/auro-kit from list ([8ca8c84](https://github.com/AlaskaAirlines/auro-kit/commit/8ca8c843cd778c3aab47045f616e4007f7e86923))

blackfalcon committed:
perf: update versions of dependencies

semantic-release-bot committed:
chore(release): 1.0.0-beta.4 [skip ci]

# [1.0.0-beta.4](https://github.com/AlaskaAirlines/auro-kit/compare/v1.0.0-beta.3...v1.0.0-beta.4) (2024-06-04)

### Performance Improvements

* update versions of dependencies ([a94fd42](https://github.com/AlaskaAirlines/auro-kit/commit/a94fd42788cb3189b303c1b641d6dee06f205031))

blackfalcon committed:
refactor: update project name

This commit updates the project name from auro-kit to aurokit as not to
be mistaken as a web component, but as an individual install tool.

On branch dsande/generalImprovements

blackfalcon committed:
perf: update docs and npm update support

This commit adds a new README to the project that better defines the
scope and intent of what AuroKit is and how it will support consuming
developers.

Additionally there is an update to the ./package.json file that replaces
the `^` with `~` to restrict `npm update` to only PATCH releases.

On branch dsande/generalImprovements
Changes to be committed:
modified:   README.md
new file:   package-lock.json
modified:   package.json

blackfalcon committed:
perf: update all packages

This commit is a complete set of auro elements that all install without
conflicts or dependency issues.

blackfalcon committed:
feat: install Storybook

This commit adds Storybook as a capability of AuroKit and begins to
define component stories.

blackfalcon committed:
perf: update to official Auro FormKit release

This commit updates to the currently official release of Auro FormKit
and the latest version of Design Tokens.

All components in these versions are stable.

blackfalcon committed:
perf: add a11y testing

This commit adds a11y testing and addresses a discovered issue with the
button story.

On branch dsande/generalImprovements
Changes to be committed:
modified:   .storybook/main.js
modified:   package-lock.json
modified:   package.json
modified:   stories/auro-button.stories.js

blackfalcon committed:
chore: update auro packages

blackfalcon committed:
docs: update README

This commit updates the README to include information regarding the
maintenance policy of the AuroKit project.

semantic-release-bot committed:
chore(release): 1.0.0-beta.5 [skip ci]

# [1.0.0-beta.5](https://github.com/AlaskaAirlines/aurokit/compare/v1.0.0-beta.4...v1.0.0-beta.5) (2025-03-07)

### Features

* install Storybook ([e12dd5b](https://github.com/AlaskaAirlines/aurokit/commit/e12dd5b093667ffe216d832f47f95cd3c44f7443))

### Performance Improvements

* add a11y testing ([7c0cba1](https://github.com/AlaskaAirlines/aurokit/commit/7c0cba162430faa61eb8fec3c4d0d5303e6a0c69))
* update all packages ([3c1bfe2](https://github.com/AlaskaAirlines/aurokit/commit/3c1bfe277c8220841db801882a2e097988d554b6))
* update docs and npm update support ([b93fe43](https://github.com/AlaskaAirlines/aurokit/commit/b93fe436c26a87418675afa7b0df65a9c48b2a1b))
* update to official Auro FormKit release ([b8e7216](https://github.com/AlaskaAirlines/aurokit/commit/b8e7216f453f86967bab28d54d3d0f8320684966))

blackfalcon committed:
feat: update auro package dependencies

This commit updates multiple Auro package dependencies to include a
number of patch and feature and updated one major release.

Changes to be committed:
modified:   package-lock.json
modified:   package.json

semantic-release-bot committed:
chore(release): 1.0.0-beta.6 [skip ci]

# [1.0.0-beta.6](https://github.com/AlaskaAirlines/aurokit/compare/v1.0.0-beta.5...v1.0.0-beta.6) (2025-03-24)

### Features

* update auro package dependencies ([5e20444](https://github.com/AlaskaAirlines/aurokit/commit/5e204448f50979ad5c4dce8c4cfccd5ed4c7b5fe))

blackfalcon committed:
feat: update auro dependencies

See the following package updates

-    "@aurodesignsystem/auro-accordion": "~5.0.0",
+    "@aurodesignsystem/auro-accordion": "~5.0.1",

-    "@aurodesignsystem/auro-button": "~9.0.0",
+    "@aurodesignsystem/auro-button": "~9.1.1",

-    "@aurodesignsystem/auro-drawer": "~4.0.1",
+    "@aurodesignsystem/auro-drawer": "~4.0.2",

-    "@aurodesignsystem/auro-formkit": "~2.0.2",
+    "@aurodesignsystem/auro-formkit": "~2.2.0",

-    "@aurodesignsystem/auro-library": "~3.0.13",
+    "@aurodesignsystem/auro-library": "~4.1.0",

-    "@aurodesignsystem/auro-nav": "~3.0.0",
+    "@aurodesignsystem/auro-nav": "~3.1.0",

On branch beta
Your branch is up to date with 'origin/beta'.
Changes to be committed:
modified:   package-lock.json
modified:   package.json

semantic-release-bot committed:
chore(release): 1.0.0-beta.7 [skip ci]

# [1.0.0-beta.7](https://github.com/AlaskaAirlines/aurokit/compare/v1.0.0-beta.6...v1.0.0-beta.7) (2025-04-02)

### Features

* update auro dependencies ([cfe8f28](https://github.com/AlaskaAirlines/aurokit/commit/cfe8f28c82164af30e866bbedc037cb3069aa0f5))

blackfalcon committed:
perf: update patch dependencies

Changes to be committed:
modified:   package-lock.json

blackfalcon committed:
feat: update feature release Auro dependencies

Changes to be committed:
modified:   package-lock.json
modified:   package.json

blackfalcon committed:
feat: update major Auro released dependencies

BREAKING CHANGE: This update includes MAJOR release updates from
flightline, formkit, icon and loader.

Changes to be committed:
modified:   package-lock.json
modified:   package.json

semantic-release-bot committed:
chore(release): 1.0.0-beta.8 [skip ci]

# [1.0.0-beta.8](https://github.com/AlaskaAirlines/aurokit/compare/v1.0.0-beta.7...v1.0.0-beta.8) (2025-04-29)

### Features

* update feature release Auro dependencies ([0d6257e](https://github.com/AlaskaAirlines/aurokit/commit/0d6257e8f3a5ea3d425d868348e08717664d43fa))
* update major Auro released dependencies ([8f32e49](https://github.com/AlaskaAirlines/aurokit/commit/8f32e49bff38b6cc5a775194a93e74a35bd85d76))

### Performance Improvements

* update patch dependencies ([8108254](https://github.com/AlaskaAirlines/aurokit/commit/8108254c233c41d3a13768182664d03035edfd54))

### BREAKING CHANGES

* This update includes MAJOR release updates from
flightline, formkit, icon and loader.

Changes to be committed:
modified:   package-lock.json
modified:   package.json

blackfalcon committed:
fix: correct dependency update type

Changes to be committed:
modified:   package-lock.json
modified:   package.json

semantic-release-bot committed:
chore(release): 1.0.0-beta.9 [skip ci]

# [1.0.0-beta.9](https://github.com/AlaskaAirlines/aurokit/compare/v1.0.0-beta.8...v1.0.0-beta.9) (2025-04-29)

### Bug Fixes

* correct dependency update type ([dfdcb9b](https://github.com/AlaskaAirlines/aurokit/commit/dfdcb9b5c41d9e41626cf98380a6c3444f78f3fd))

blackfalcon committed:
chore: update Auro package dependencies

BREAKING CHANGE: The following update includes a MAJOR release.

"@aurodesignsystem/auro-badge": "~5.0.0"
https://github.com/AlaskaAirlines/auro-badge/commit/b3c9a0e807592c31d959309626a8ab5938403899

Changes to be committed:
modified:   package-lock.json
modified:   package.json

semantic-release-bot committed:
chore(release): 1.0.0-beta.10 [skip ci]

# [1.0.0-beta.10](https://github.com/AlaskaAirlines/aurokit/compare/v1.0.0-beta.9...v1.0.0-beta.10) (2025-05-09)

### chore

* update Auro package dependencies ([7508db0](https://github.com/AlaskaAirlines/aurokit/commit/7508db0c1aedce5293d3ecbfe68e69cea88140bd))

### BREAKING CHANGES

* The following update includes a MAJOR release.

"@aurodesignsystem/auro-badge": "~5.0.0"
https://github.com/AlaskaAirlines/auro-badge/commit/b3c9a0e807592c31d959309626a8ab5938403899

Changes to be committed:
modified:   package-lock.json
modified:   package.json

blackfalcon committed:
feat: update MINOR versions of Auro dependencies

The following Auro dependencies were updates with this release.

-    "@alaskaairux/icons": "~5.3.0",
-    "@aurodesignsystem/auro-accordion": "~5.1.3",
+    "@alaskaairux/icons": "~5.4.0",
+    "@aurodesignsystem/auro-accordion": "~5.2.1",

-    "@aurodesignsystem/auro-button": "~9.3.3",
+    "@aurodesignsystem/auro-button": "~9.4.0",

-    "@aurodesignsystem/auro-formkit": "~3.2.8",
+    "@aurodesignsystem/auro-formkit": "~3.5.0",

-    "@aurodesignsystem/auro-library": "~4.4.1",
+    "@aurodesignsystem/auro-library": "~4.5.0",

-    "@aurodesignsystem/design-tokens": "~5.14.0",
-    "@aurodesignsystem/webcorestylesheets": "~6.5.0",
+    "@aurodesignsystem/design-tokens": "~5.15.1",
+    "@aurodesignsystem/webcorestylesheets": "~6.6.2",

On branch beta
Your branch is up to date with 'origin/beta'.

Changes to be committed:
modified:   package-lock.json
modified:   package.json

semantic-release-bot committed:
chore(release): 1.0.0-beta.11 [skip ci]

# [1.0.0-beta.11](https://github.com/AlaskaAirlines/aurokit/compare/v1.0.0-beta.10...v1.0.0-beta.11) (2025-05-23)

### Features

* update MINOR versions of Auro dependencies ([bd51d1e](https://github.com/AlaskaAirlines/aurokit/commit/bd51d1e1eb3e2bec7520cd0f2666dcdf16b83d74))

blackfalcon committed:
docs: update README with npm update info

On branch beta
Your branch is up to date with 'origin/beta'.
Changes to be committed:
modified:   README.md

blackfalcon committed:
feat: update Auro dependencies

BREAKING CHANGE: This commit includes MAJOR updates to some packages.

-    "@aurodesignsystem/auro-carousel": "~4.0.2",
+    "@aurodesignsystem/auro-carousel": "~4.1.0",

-    "@aurodesignsystem/auro-formkit": "~3.5.0",
+    "@aurodesignsystem/auro-formkit": "~4.0.0",

-    "@aurodesignsystem/auro-hyperlink": "~5.1.2",
+    "@aurodesignsystem/auro-hyperlink": "~5.2.0",

-    "@aurodesignsystem/auro-library": "~4.5.0",
+    "@aurodesignsystem/auro-library": "~5.0.2",

-    "@aurodesignsystem/design-tokens": "~5.15.1",
-    "@aurodesignsystem/webcorestylesheets": "~6.6.2",
+    "@aurodesignsystem/design-tokens": "~8.1.0",
+    "@aurodesignsystem/webcorestylesheets": "~8.0.1",

blackfalcon committed:
docs: update readme

Add information about how to manage overrides.

Changes to be committed:
modified:   README.md

semantic-release-bot committed:
chore(release): 1.0.0-beta.12 [skip ci]

# [1.0.0-beta.12](https://github.com/AlaskaAirlines/aurokit/compare/v1.0.0-beta.11...v1.0.0-beta.12) (2025-06-05)

### Features

* update Auro dependencies ([893d0cf](https://github.com/AlaskaAirlines/aurokit/commit/893d0cfefb91da2ef4b4a827dafa2d9a019305e0))

### BREAKING CHANGES

* This commit includes MAJOR updates to some packages.

-    "@aurodesignsystem/auro-carousel": "~4.0.2",
+    "@aurodesignsystem/auro-carousel": "~4.1.0",

-    "@aurodesignsystem/auro-formkit": "~3.5.0",
+    "@aurodesignsystem/auro-formkit": "~4.0.0",

-    "@aurodesignsystem/auro-hyperlink": "~5.1.2",
+    "@aurodesignsystem/auro-hyperlink": "~5.2.0",

-    "@aurodesignsystem/auro-library": "~4.5.0",
+    "@aurodesignsystem/auro-library": "~5.0.2",

-    "@aurodesignsystem/design-tokens": "~5.15.1",
-    "@aurodesignsystem/webcorestylesheets": "~6.6.2",
+    "@aurodesignsystem/design-tokens": "~8.1.0",
+    "@aurodesignsystem/webcorestylesheets": "~8.0.1",

blackfalcon committed:
perf: update MINOR dependency packages

-    "@alaskaairux/icons": "~5.4.0",
+    "@alaskaairux/icons": "~5.5.0",

-    "@aurodesignsystem/design-tokens": "~8.1.0",
-    "@aurodesignsystem/webcorestylesheets": "~8.0.1",
+    "@aurodesignsystem/design-tokens": "~8.2.0",
+    "@aurodesignsystem/webcorestylesheets": "~8.1.0",

e
Changes to be committed:
modified:   package-lock.json
modified:   package.json

semantic-release-bot committed:
chore(release): 1.0.0-beta.13 [skip ci]

# [1.0.0-beta.13](https://github.com/AlaskaAirlines/aurokit/compare/v1.0.0-beta.12...v1.0.0-beta.13) (2025-06-09)

### Performance Improvements

* update MINOR dependency packages ([ad14028](https://github.com/AlaskaAirlines/aurokit/commit/ad14028e894a164483c5d7f82a56507bc4fc527e))

blackfalcon committed:
fix: move dependency to devDependency

The dependency, @storybook/addon-a11y, was moved from a standard
dependency to devDependency to address Aquascan alert.

Changes to be committed:
modified:   package-lock.json
modified:   package.json

semantic-release-bot committed:
chore(release): 1.0.0-beta.14 [skip ci]

# [1.0.0-beta.14](https://github.com/AlaskaAirlines/aurokit/compare/v1.0.0-beta.13...v1.0.0-beta.14) (2025-06-18)

### Bug Fixes

* move dependency to devDependency ([3ee751b](https://github.com/AlaskaAirlines/aurokit/commit/3ee751b6181870e26d434632074ce43ca32c73b3))

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/AlaskaAirlines/AuroKit?pullRequest=4) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/AlaskaAirlines/AuroKit?pullRequest=4) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: blackfalcon<br/>:x: semantic-release-bot<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/AlaskaAirlines/AuroKit?pullRequest=4) it.</sub>

blackfalcon committed:
perf: updates per sourcery-ai feedback

Changes to be committed:
modified:   .github/workflows/testPublish.yml
modified:   README.md

semantic-release-bot committed:
chore(release): 1.0.0-beta.1 [skip ci]

# 1.0.0-beta.1 (2025-06-18)

### Bug Fixes

* correct dependency update type ([139ae4b](https://github.com/AlaskaAirlines/aurokit/commit/139ae4b9acaa13860b1be418eed76333277af0c8))
* move dependency to devDependency ([a3219ac](https://github.com/AlaskaAirlines/aurokit/commit/a3219ac076b9d4736d2b5132ad43b9b2e9b81c5e))

### chore

* update Auro package dependencies ([b7b8d5c](https://github.com/AlaskaAirlines/aurokit/commit/b7b8d5c100348c364cbaeb83ebd62a89a22a5713))

### Features

* install Storybook ([03e1498](https://github.com/AlaskaAirlines/aurokit/commit/03e149841892bbe8f23f5ac877890b0580e682a0))
* update auro dependencies ([0898d51](https://github.com/AlaskaAirlines/aurokit/commit/0898d5198307580efb342f70b6616599c26a0345))
* update Auro dependencies ([256c3ca](https://github.com/AlaskaAirlines/aurokit/commit/256c3ca13198870f73b8418b9f18a94891ba778b))
* update auro package dependencies ([1308675](https://github.com/AlaskaAirlines/aurokit/commit/130867553db303464c6dba15d5e60edcb1a05584))
* update feature release Auro dependencies ([aedd78c](https://github.com/AlaskaAirlines/aurokit/commit/aedd78caad0ada451f32a020f3cde7841e95f5c5))
* update major Auro released dependencies ([638032e](https://github.com/AlaskaAirlines/aurokit/commit/638032eb83ffb5236acdaaca3f282aaa03362eca))
* update MINOR versions of Auro dependencies ([c944428](https://github.com/AlaskaAirlines/aurokit/commit/c94442889981005322e8fa511d0b32b1e2ffedae))

### Performance Improvements

* add a11y testing ([77bda2c](https://github.com/AlaskaAirlines/aurokit/commit/77bda2c8317e2d325598c8c043f35b360a0d99f6))
* add accordion to package.json ([38256f2](https://github.com/AlaskaAirlines/aurokit/commit/38256f21e6e966f51504b9de955932f5e426cfea))
* add remaining Auro elements ([f4d103f](https://github.com/AlaskaAirlines/aurokit/commit/f4d103f7095b6aed48dc6defc4ca2253088f8646))
* remove @aurodesignsystem/auro-kit from list ([2914d98](https://github.com/AlaskaAirlines/aurokit/commit/2914d98f11cd2d72a45c5cd599cdced47bb69758))
* update all packages ([6ca8a59](https://github.com/AlaskaAirlines/aurokit/commit/6ca8a59a487cfe9dc2ddec9cce7e053d16c50b55))
* update docs and npm update support ([9c10248](https://github.com/AlaskaAirlines/aurokit/commit/9c10248362e91651fc7d8dcf7b37b172c84fe3a2))
* update MINOR dependency packages ([104c3a4](https://github.com/AlaskaAirlines/aurokit/commit/104c3a43f7abfddcc5574c242d73acffad31caec))
* update patch dependencies ([b23cbe2](https://github.com/AlaskaAirlines/aurokit/commit/b23cbe2dfab9e4164e4d1c4af9aa3a10d2abd5c9))
* update to official Auro FormKit release ([50aa7bc](https://github.com/AlaskaAirlines/aurokit/commit/50aa7bcb05a911d31d67a3073fde27e22ae521ab))
* update versions of dependencies ([99f4dca](https://github.com/AlaskaAirlines/aurokit/commit/99f4dca55d3ff122759e713e0715acb48f497976))
* updates per sourcery-ai feedback ([3a08364](https://github.com/AlaskaAirlines/aurokit/commit/3a0836471aaf1a3b71b148134cfa43f2a176b733))

### BREAKING CHANGES

* This commit includes MAJOR updates to some packages.

-    "@aurodesignsystem/auro-carousel": "~4.0.2",
+    "@aurodesignsystem/auro-carousel": "~4.1.0",

-    "@aurodesignsystem/auro-formkit": "~3.5.0",
+    "@aurodesignsystem/auro-formkit": "~4.0.0",

-    "@aurodesignsystem/auro-hyperlink": "~5.1.2",
+    "@aurodesignsystem/auro-hyperlink": "~5.2.0",

-    "@aurodesignsystem/auro-library": "~4.5.0",
+    "@aurodesignsystem/auro-library": "~5.0.2",

-    "@aurodesignsystem/design-tokens": "~5.15.1",
-    "@aurodesignsystem/webcorestylesheets": "~6.6.2",
+    "@aurodesignsystem/design-tokens": "~8.1.0",
+    "@aurodesignsystem/webcorestylesheets": "~8.0.1",
* The following update includes a MAJOR release.

"@aurodesignsystem/auro-badge": "~5.0.0"
https://github.com/AlaskaAirlines/auro-badge/commit/b3c9a0e807592c31d959309626a8ab5938403899

Changes to be committed:
modified:   package-lock.json
modified:   package.json
* This update includes MAJOR release updates from
flightline, formkit, icon and loader.

Changes to be committed:
modified:   package-lock.json
modified:   package.json

blackfalcon committed:
fix: update beta version

Changes to be committed:
modified:   package-lock.json
modified:   package.json

semantic-release-bot committed:
chore(release): 1.0.0-beta.1 [skip ci]

# 1.0.0-beta.1 (2025-06-18)

### Bug Fixes

* correct dependency update type ([139ae4b](https://github.com/AlaskaAirlines/aurokit/commit/139ae4b9acaa13860b1be418eed76333277af0c8))
* move dependency to devDependency ([a3219ac](https://github.com/AlaskaAirlines/aurokit/commit/a3219ac076b9d4736d2b5132ad43b9b2e9b81c5e))
* update beta version ([c3f0358](https://github.com/AlaskaAirlines/aurokit/commit/c3f03588e69226e2f8e637dd421c303ade6cdaa3))

### chore

* update Auro package dependencies ([b7b8d5c](https://github.com/AlaskaAirlines/aurokit/commit/b7b8d5c100348c364cbaeb83ebd62a89a22a5713))

### Features

* install Storybook ([03e1498](https://github.com/AlaskaAirlines/aurokit/commit/03e149841892bbe8f23f5ac877890b0580e682a0))
* update auro dependencies ([0898d51](https://github.com/AlaskaAirlines/aurokit/commit/0898d5198307580efb342f70b6616599c26a0345))
* update Auro dependencies ([256c3ca](https://github.com/AlaskaAirlines/aurokit/commit/256c3ca13198870f73b8418b9f18a94891ba778b))
* update auro package dependencies ([1308675](https://github.com/AlaskaAirlines/aurokit/commit/130867553db303464c6dba15d5e60edcb1a05584))
* update feature release Auro dependencies ([aedd78c](https://github.com/AlaskaAirlines/aurokit/commit/aedd78caad0ada451f32a020f3cde7841e95f5c5))
* update major Auro released dependencies ([638032e](https://github.com/AlaskaAirlines/aurokit/commit/638032eb83ffb5236acdaaca3f282aaa03362eca))
* update MINOR versions of Auro dependencies ([c944428](https://github.com/AlaskaAirlines/aurokit/commit/c94442889981005322e8fa511d0b32b1e2ffedae))

### Performance Improvements

* add a11y testing ([77bda2c](https://github.com/AlaskaAirlines/aurokit/commit/77bda2c8317e2d325598c8c043f35b360a0d99f6))
* add accordion to package.json ([38256f2](https://github.com/AlaskaAirlines/aurokit/commit/38256f21e6e966f51504b9de955932f5e426cfea))
* add remaining Auro elements ([f4d103f](https://github.com/AlaskaAirlines/aurokit/commit/f4d103f7095b6aed48dc6defc4ca2253088f8646))
* remove @aurodesignsystem/auro-kit from list ([2914d98](https://github.com/AlaskaAirlines/aurokit/commit/2914d98f11cd2d72a45c5cd599cdced47bb69758))
* update all packages ([6ca8a59](https://github.com/AlaskaAirlines/aurokit/commit/6ca8a59a487cfe9dc2ddec9cce7e053d16c50b55))
* update docs and npm update support ([9c10248](https://github.com/AlaskaAirlines/aurokit/commit/9c10248362e91651fc7d8dcf7b37b172c84fe3a2))
* update MINOR dependency packages ([104c3a4](https://github.com/AlaskaAirlines/aurokit/commit/104c3a43f7abfddcc5574c242d73acffad31caec))
* update patch dependencies ([b23cbe2](https://github.com/AlaskaAirlines/aurokit/commit/b23cbe2dfab9e4164e4d1c4af9aa3a10d2abd5c9))
* update to official Auro FormKit release ([50aa7bc](https://github.com/AlaskaAirlines/aurokit/commit/50aa7bcb05a911d31d67a3073fde27e22ae521ab))
* update versions of dependencies ([99f4dca](https://github.com/AlaskaAirlines/aurokit/commit/99f4dca55d3ff122759e713e0715acb48f497976))
* updates per sourcery-ai feedback ([3a08364](https://github.com/AlaskaAirlines/aurokit/commit/3a0836471aaf1a3b71b148134cfa43f2a176b733))

### BREAKING CHANGES

* This commit includes MAJOR updates to some packages.

-    "@aurodesignsystem/auro-carousel": "~4.0.2",
+    "@aurodesignsystem/auro-carousel": "~4.1.0",

-    "@aurodesignsystem/auro-formkit": "~3.5.0",
+    "@aurodesignsystem/auro-formkit": "~4.0.0",

-    "@aurodesignsystem/auro-hyperlink": "~5.1.2",
+    "@aurodesignsystem/auro-hyperlink": "~5.2.0",

-    "@aurodesignsystem/auro-library": "~4.5.0",
+    "@aurodesignsystem/auro-library": "~5.0.2",

-    "@aurodesignsystem/design-tokens": "~5.15.1",
-    "@aurodesignsystem/webcorestylesheets": "~6.6.2",
+    "@aurodesignsystem/design-tokens": "~8.1.0",
+    "@aurodesignsystem/webcorestylesheets": "~8.0.1",
* The following update includes a MAJOR release.

"@aurodesignsystem/auro-badge": "~5.0.0"
https://github.com/AlaskaAirlines/auro-badge/commit/b3c9a0e807592c31d959309626a8ab5938403899

Changes to be committed:
modified:   package-lock.json
modified:   package.json
* This update includes MAJOR release updates from
flightline, formkit, icon and loader.

Changes to be committed:
modified:   package-lock.json
modified:   package.json

blackfalcon committed:
fix: update package version

semantic-release-bot committed:
chore(release): 1.0.0-beta.1 [skip ci]

# 1.0.0-beta.1 (2025-06-18)

### Bug Fixes

* correct dependency update type ([139ae4b](https://github.com/AlaskaAirlines/aurokit/commit/139ae4b9acaa13860b1be418eed76333277af0c8))
* move dependency to devDependency ([a3219ac](https://github.com/AlaskaAirlines/aurokit/commit/a3219ac076b9d4736d2b5132ad43b9b2e9b81c5e))
* update beta version ([c3f0358](https://github.com/AlaskaAirlines/aurokit/commit/c3f03588e69226e2f8e637dd421c303ade6cdaa3))
* update package version ([7f4a3b8](https://github.com/AlaskaAirlines/aurokit/commit/7f4a3b8161c184458d8429fe7df314379ca23f51))

### chore

* update Auro package dependencies ([b7b8d5c](https://github.com/AlaskaAirlines/aurokit/commit/b7b8d5c100348c364cbaeb83ebd62a89a22a5713))

### Features

* install Storybook ([03e1498](https://github.com/AlaskaAirlines/aurokit/commit/03e149841892bbe8f23f5ac877890b0580e682a0))
* update auro dependencies ([0898d51](https://github.com/AlaskaAirlines/aurokit/commit/0898d5198307580efb342f70b6616599c26a0345))
* update Auro dependencies ([256c3ca](https://github.com/AlaskaAirlines/aurokit/commit/256c3ca13198870f73b8418b9f18a94891ba778b))
* update auro package dependencies ([1308675](https://github.com/AlaskaAirlines/aurokit/commit/130867553db303464c6dba15d5e60edcb1a05584))
* update feature release Auro dependencies ([aedd78c](https://github.com/AlaskaAirlines/aurokit/commit/aedd78caad0ada451f32a020f3cde7841e95f5c5))
* update major Auro released dependencies ([638032e](https://github.com/AlaskaAirlines/aurokit/commit/638032eb83ffb5236acdaaca3f282aaa03362eca))
* update MINOR versions of Auro dependencies ([c944428](https://github.com/AlaskaAirlines/aurokit/commit/c94442889981005322e8fa511d0b32b1e2ffedae))

### Performance Improvements

* add a11y testing ([77bda2c](https://github.com/AlaskaAirlines/aurokit/commit/77bda2c8317e2d325598c8c043f35b360a0d99f6))
* add accordion to package.json ([38256f2](https://github.com/AlaskaAirlines/aurokit/commit/38256f21e6e966f51504b9de955932f5e426cfea))
* add remaining Auro elements ([f4d103f](https://github.com/AlaskaAirlines/aurokit/commit/f4d103f7095b6aed48dc6defc4ca2253088f8646))
* remove @aurodesignsystem/auro-kit from list ([2914d98](https://github.com/AlaskaAirlines/aurokit/commit/2914d98f11cd2d72a45c5cd599cdced47bb69758))
* update all packages ([6ca8a59](https://github.com/AlaskaAirlines/aurokit/commit/6ca8a59a487cfe9dc2ddec9cce7e053d16c50b55))
* update docs and npm update support ([9c10248](https://github.com/AlaskaAirlines/aurokit/commit/9c10248362e91651fc7d8dcf7b37b172c84fe3a2))
* update MINOR dependency packages ([104c3a4](https://github.com/AlaskaAirlines/aurokit/commit/104c3a43f7abfddcc5574c242d73acffad31caec))
* update patch dependencies ([b23cbe2](https://github.com/AlaskaAirlines/aurokit/commit/b23cbe2dfab9e4164e4d1c4af9aa3a10d2abd5c9))
* update to official Auro FormKit release ([50aa7bc](https://github.com/AlaskaAirlines/aurokit/commit/50aa7bcb05a911d31d67a3073fde27e22ae521ab))
* update versions of dependencies ([99f4dca](https://github.com/AlaskaAirlines/aurokit/commit/99f4dca55d3ff122759e713e0715acb48f497976))
* updates per sourcery-ai feedback ([3a08364](https://github.com/AlaskaAirlines/aurokit/commit/3a0836471aaf1a3b71b148134cfa43f2a176b733))

### BREAKING CHANGES

* This commit includes MAJOR updates to some packages.

-    "@aurodesignsystem/auro-carousel": "~4.0.2",
+    "@aurodesignsystem/auro-carousel": "~4.1.0",

-    "@aurodesignsystem/auro-formkit": "~3.5.0",
+    "@aurodesignsystem/auro-formkit": "~4.0.0",

-    "@aurodesignsystem/auro-hyperlink": "~5.1.2",
+    "@aurodesignsystem/auro-hyperlink": "~5.2.0",

-    "@aurodesignsystem/auro-library": "~4.5.0",
+    "@aurodesignsystem/auro-library": "~5.0.2",

-    "@aurodesignsystem/design-tokens": "~5.15.1",
-    "@aurodesignsystem/webcorestylesheets": "~6.6.2",
+    "@aurodesignsystem/design-tokens": "~8.1.0",
+    "@aurodesignsystem/webcorestylesheets": "~8.0.1",
* The following update includes a MAJOR release.

"@aurodesignsystem/auro-badge": "~5.0.0"
https://github.com/AlaskaAirlines/auro-badge/commit/b3c9a0e807592c31d959309626a8ab5938403899

Changes to be committed:
modified:   package-lock.json
modified:   package.json
* This update includes MAJOR release updates from
flightline, formkit, icon and loader.

Changes to be committed:
modified:   package-lock.json
modified:   package.json
